### ISSUE TITLE
Incremental Training ?

### ISSUE THREAD

**Issue by cetinol** [arahusky/Tensorflow-Segmentation#13]
Thank you very much for this inspiring project.

This is not an issue but this is the only way I could manage to get in touch with you, sorry for the inconvenience.
İs there a way to train the model incrementally  or each time should I start from the beginning  ?

**Comment by rxu14** [comment]
Hey,

I am not the author, but you could always save mode with a large learning rate at beginning, then stop train it .Then  decrease learning rate and load trained model using tf.restore and continue trainning.

**Comment by cetinol** [comment]
Thank you for the reply

On Jan 31, 2018 8:36 AM, "Runsheng Xu" <notifications@github.com> wrote:

Hey,

I am not the author, but you could always save mode with a large learning
rate at beginning, then stop train it .Then decrease learning rate and load
trained model using tf.restore and continue trainning.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/arahusky/Tensorflow-Segmentation/issues/13#issuecomment-361829219>,
or mute the thread

**Comment by arahusky** [comment]
Hello,
as @rxu14 says (thank you for it), you can stop training at particular moment (save the model weights), restore model weights:

```
ckpt = tf.train.get_checkpoint_state(checkpoint)
if ckpt and ckpt.model_checkpoint_path:
    saver.restore(session, ckpt.model_checkpoint_path)
```

and run new training (with possibly lower learning rate). You would have to, however, impement this yourself as this is not present in this simple demo.
